examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/transformers/weight_only/text-generation/run_generation_cpu_woq.py

```diff
@@ -78,9 +79,14 @@
 parser.add_argument("--group_size", type=int, default=128)
 parser.add_argument("--scheme", default=None)
 parser.add_argument(
-    "--layer_wise",
-    action="store_true",
-    help="Use layer wise to do quantization",
+    "--use_layer_wise",
+    nargs='?', 
+    const=True,
+    default=None,
+    type=lambda x: bool(strtobool(x)), 
+    help="""Use layer wise to do quantization. Case-insensitive and
```

Comment:
May add explain for "None", which means it will be determined automatically if the platform is client